solution.sh

```diff
@@ -0,0 +1,13 @@
+rm -r mydir
+mkdir mydir
+cd mydir
+mkdir mydir2
+cd mydir2
+touch myfile
+echo "Hello bash" > myfile
+cat myfile
+cd ..
+cp -r mydir2 mydir3
+find .| sort -r
+
+#the first line is so it doesnt make multiple files and directories
```

Comment:
Good idea, in real practice this would be done as part of a Makefile, for example:

```
run: clean
    ./solution.sh

clean:
    rm -r mydir
```

Then you would run with `make run`